source/mouseHandler.py

```diff
@@ -379,3 +379,32 @@ def unlockRightMouseButton():
 	# Translators: This is presented when the right mouse button lock is released (used for drag and drop).
 	ui.message(_("Right mouse button unlock"))
 	executeMouseEvent(winUser.MOUSEEVENTF_RIGHTUP, 0, 0)
+
+
+def scrollMouseWheel(scrollSteps: int, isVertical: bool = True) -> None:
+	"""
+	Scrolls the mouse wheel either vertically or horizontally, controlling scroll direction and amount.
+	Consult https://learn.microsoft.com/en-us/windows/win32/inputdev/wm-mousewheel for more about mouse events.
+	@param scrollSteps: The number of steps to scroll. Each step should
+		 correspond to a fraction or multiple of WHEEL_DELTA, which is typically set to 120.
+		 This defines the standard increment or decrement in scroll position.
+	@type scrollSteps: int

```

Comment:
You added type hints, so I think specifying types in the doc string is not necessary.
Also make sure to make the parameters sphinx style.